### PR TITLE
hotfix - bad distance sensor example

### PR DESCRIPTION
**Pull request by RobertLucian**
This PR solves the following issue: RobertLucian/GoPiGo3#13.

### CODE REVIEW

File: Software/Python/Examples/easy_Distance_Sensor.py [hunk 1]
```diff
@@ -32,8 +32,16 @@
...
+
+# Create an instance of the GoPiGo3 class.
+# GPG will be the GoPiGo3 object.
+gpg = easy.EasyGoPiGo3()
+
 # Create an instance of the Distance Sensor class.
-my_distance_sensor = easy.DistanceSensor()     # Distance_Sensor will be the Line Follower object.
+# I2C1 and I2C2 are just labels used for identifyng the port on the GoPiGo3 board.
+# But technically, I2C1 and I2C2 are the same thing, so we don't have to pass any port to the constructor.
+my_distance_sensor = gpg.init_distance_sensor()
```
Comment (johnisanerd) [inline Software/Python/Examples/easy_Distance_Sensor.py]: Hey @RobertLucian when I run this new example on DexterOS I still get the following error thrown; could it be an issue with DexterOS we need to dig into?

` sudo python easy_Distance_Sensor.py
Loading library without distance sensor
Distance Sensor init: class DistanceSensor has no attribute '__init__'
Traceback (most recent call last):
  File "easy_Distance_Sensor.py", line 44, in <module>
    my_distance_sensor = gpg.init_distance_sensor()
  File "build/bdist.linux-armv7l/egg/easygopigo3.py", line 839, in init_distance_sensor
  File "build/bdist.linux-armv7l/egg/easygopigo3.py", line 2615, in __init__
AttributeError: class DistanceSensor has no attribute '__init__'
`

Comment (RobertLucian) [reply Software/Python/Examples/easy_Distance_Sensor.py]: It seems like the `DI_Sensors` is not installed. 
To see if you have the distance sensor installed, open a `Python` terminal and type `from di_sensors import distance_sensor`. If it's an error, then you don't have `DI_Sensors` installed.

Also, though I'm not sure if this matters, are you on a GoPiGo3 or an a GoPiGo2?

Just for the records, on my GoPiGo3 it works.

Comment (johnisanerd) [reply Software/Python/Examples/easy_Distance_Sensor.py]: Good points all around.  I'm using the GPG3 for sure.  

I can't import distance_sensor, that throws the "no module named di_sensors" error.

This is on DexterOS though; @CleoQc should that be a problem?  I'm running v1.1.0Etha

Comment (RobertLucian) [reply Software/Python/Examples/easy_Distance_Sensor.py]: Hmm, `v1.1.0Etha` already has `DI_Sensors` in it. Can you redo the update and show the system logs?

**Pull request merged**